.gitignore

```diff
@@ -3,3 +3,4 @@
 .tours/codeql-tutorial-database/db-csv/default/strings/
 .cache/
 
+tutorial-queries/codeql-pack.lock.yml
```

Comment:
I assume this is okay? This lock file gets generated by the `pack install` command, and I don't think we want to track it in Git.

Reply:
Yeah, that makes sense. We don't want to confuse the user as to why their fresh codespace has uncommitted changes.

Reply:
I would actually generate this lock file now and commit it to Git. Then the `pack install` will use the existing lock file.

Reply:
That makes sense! I don't expect that the versions will change very often anyway 😄